mezzo/server/spotify_api.js

```diff
@@ -19,33 +19,30 @@ async function fetchToken () {
   return token_object.data.access_token;
 }
 
-spotify.post('/refresh_token', async (req, res) => {
-  try {
-    spotifyToken = spotifyToken + await fetchToken();
-    spotifyToken = "Bearer " + spotifyToken;
-    console.log(spotifyToken);
-    res.send("Token Granted");
-  } catch (err) {
-    console.log(err);
-  }
-});
-
-spotify.get('/home', async (req, res) => {
-  spotifyToken = spotifyToken + await fetchToken();
+spotify.post('/api/getSearchResults', async (req, res) => {
```

Comment:
You don't need /api/getSearchResults here... since it is using the spotify. prefix, it can just be '/getSearchResults'

Reply:
I did intentionally put api, but I can omit it